spec/policies/agent/rdv_policy_spec.rb

```diff
@@ -119,7 +119,7 @@
       rdv.participations.build(user_id: user_from_other_territory.id, created_by: agent)
     end
 
-    it_behaves_like "permit actions", :rdv, :destroy?
+    it_behaves_like "permit actions", :rdv, :show?, :destroy?
     it_behaves_like "not permit actions", :rdv, :new?, :create?, :edit?
```

Comment:
on peut éventuellement rajouter `update?` ici pour être plus explicite ?

```suggestion
    it_behaves_like "not permit actions", :rdv, :new?, :create?, :edit?, :update?
```

Reply:
Très bonne suggestion, c'est commité !